File: Fantasy_Football/Models/HotCold.cs
namespace Fantasy_Football.Models
{
    
        public class HotCold
        {
            public Most_Added[] most_added { get; set; }
            public Most_Dropped[] most_dropped { get; set; }
        }

        public class Most_Added
        {
            public string playerId { get; set; }
            public string name { get; set; }
            public string team { get; set; }
            public string position { get; set; }
        }

        public class Most_Dropped
        {
            public string playerId { get; set; }
            public string name { get; set; }
            public string team { get; set; }
            public string position { get; set; }
        }

    
}

File: Fantasy_Football/ClientApp/src/app/models/hot-cold.ts

    export interface HotCold {
        most_added:   Most[];
        most_dropped: Most[];
    }
    
    export interface Most {
        playerId: string;
        name:     string;
        team:     string;
        position: Position;
    }
    
    export enum Position {
        Def = "DEF",
        K = "K",
        Qb = "QB",
        Rb = "RB",
        Te = "TE",
        Wr = "WR",
    }

